src/hidra/receiver/plugins/asapo_producer.py

```diff
@@ -321,7 +321,7 @@ def _parse_file_name(self, path):
         stream = get_entry(matched, "scan_id")
         try:
             file_idx = int(get_entry(matched, "file_idx_in_scan"))
-        except Exception as e:
+        except ValueError:
```

Comment:
I think rewriting the `ValueError` message is not necessary because you never look at it? Therefore this try except block can be removed.